src/service/url-replacements-impl.js

```diff
@@ -29,6 +29,7 @@ import {viewportForDoc} from '../viewport';
 import {userNotificationManagerFor} from '../user-notification';
 import {activityFor} from '../activity';
 import {isExperimentOn} from '../experiments';
+import {trackImpressionPromise} from '../impression.js';
```

Comment:
changed to `getTrackImpressionPromise` and add the `dev().assert` inside this function